qemu/tests/balloon_check.py

```diff
@@ -243,8 +243,11 @@ def wait_for_balloon_complete(self, timeout):
         logging.info("Wait until guest memory don't change")
         threshold = int(self.params.get("guest_stable_threshold", 100))
         is_stable = self._mem_state(threshold)
-        utils_misc.wait_for(lambda: next(is_stable), timeout,
-                            step=float(self.params.get("guest_check_step", 10.0)))
+        ret = utils_misc.wait_for(lambda: next(is_stable), timeout,
+                                  step=float(self.params.get("guest_check_step",
+                                                             10.0)))
+        if not ret:
```

Comment:
I checked the return value of wait_for.  I guess it's better to use `if ret is None` here.
```
def wait_for(func, timeout, first=0.0, step=1.0, text=None):
    """
    Wait until func() evaluates to True.

    If func() evaluates to True before timeout expires, return the
    value of func(). Otherwise return None.

```

Reply:
@huangyum  Could you explain the difference ?   As my understanding,  it is same here.

Reply:
From the function description,  it will return the value of func() or None.  If the value of func() is False or 0 or '', `not ret` will be true, it's same when ret is None. I guess it's more straight forward to use the value described.

Reply:
`None`, `False`, `0`, `[]`, `{}` are all same if we use  `if ***`,   and different when use `if *** is ***`.
So let me keep it, more concise, thanks !